apache/superset
# refactor: Existing Antd Collapse to enhanced

### SUMMARY
This enhanced the Antd Collapse with Emotion styled components and replaces the existing instances of Antd Collapse with the enhanced one

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE:
<img width="349" alt="before-1" src="https://user-images.githubusercontent.com/60598000/101920429-76f7f800-3bd4-11eb-806e-eceb131be4e1.png">
AFTER:
<img width="315" alt="after-1" src="https://user-images.githubusercontent.com/60598000/101920455-7f503300-3bd4-11eb-9f95-cd66ea9b268e.png">

BEFORE:
<img width="1631" alt="before-2" src="https://user-images.githubusercontent.com/60598000/101920433-78292500-3bd4-11eb-85a8-a6518b04ebbc.png">
AFTER:
<img width="1660" alt="after-2" src="https://user-images.githubusercontent.com/60598000/101920458-7fe8c980-3bd4-11eb-8fad-5027096b121e.png">

BEFORE:
<img width="1087" alt="before-3" src="https://user-images.githubusercontent.com/60598000/101920438-78292500-3bd4-11eb-8a26-389749c895b5.png">
AFTER:
<img width="1097" alt="after-3" src="https://user-images.githubusercontent.com/60598000/101920459-80816000-3bd4-11eb-89b3-a27769892ad3.png">

### TEST PLAN
1. Log into Superset
2. Open the different collapsible sections in the welcome page
3. Go to Dashboards and select an existing dashboard
4. Check for the filter icon on top of each chart and show the filters
5. Expand a chart
6. Look for the "Data" collapse bottom right and open it

### ADDITIONAL INFORMATION
